lib/point_quest_web/live/quest.ex

```diff
@@ -22,15 +22,19 @@ defmodule PointQuestWeb.QuestLive do
           <.button phx-click="copy_link">Copy Invite Link</.button>
         </div>
       </div>
-
-      <div>
-        <pre><code><%= Jason.encode!(Ecto.embedded_dump(@quest, :json), pretty: true) %></code></pre>
-      </div>
       <div class="flex gap-4">
-        <div :for={{user_id, %{name: name, class: class}} <- @users} class="bg-blue-400">
-          <%= user_id %>
-          <%= name %>
-          <%= class %>
+        <!-- <div :for={{user_id, %{name: name, class: class}} <- @users} class="flex flex-col p-2"> -->
```

Comment:
⚒️

Reply:
Whoops, thanks. Getting sloppy in my old age...